frontend/rust-lib/flowy-user/src/entities/user_setting.rs

```diff
@@ -62,6 +70,22 @@ pub enum ThemeModePB {
   System = 2,
 }
 
+#[derive(ProtoBuf_Enum, Serialize, Deserialize, Clone, Debug, Default)]
+pub enum LayoutDirectionPB {
+  #[default]
+  LTRLayout = 0,
+  RTLLayout = 1,
+}
+
+#[derive(ProtoBuf_Enum, Serialize, Deserialize, Clone, Debug, Default)]
+pub enum TextDirectionPB {
+  LTR = 0,
+  RTL = 1,
+  AUTO = 2,
+  #[default]
+  NULL = 3,
```

Comment:
With NULL, I mean STH like not set (which we have in some other settings) and it can fallback to layout direction which is LTR by default.

Overall I'm not that sure about this. I will set it to AUTO for myself but I haven't seen AUTO as default in other cases (I think GMAIL is AUTO by default).